client/src/catalog-app.html

```diff
@@ -592,8 +592,14 @@ <h2 id="domain-title">webcomponents.org</h2>
       _notifyUser: function(event) {
         var options = event.detail;
         this.$.toast.text = options.message;
-        this.$.toastAction.textContent = options.buttonText;
+        if (options.hasOwnProperty('buttonText')) {
+          this.$.toastAction.removeAttribute('hidden');
```

Comment:
Is this necessary? Can we just change the next line to `... = options.buttonText || ''`